src/plugins/intel_npu/tools/protopipe/src/parser/config.cpp

```diff
@@ -345,6 +345,10 @@ struct convert<OpenVINOParams> {
             params.output_model_layout = node["oml"].as<LayerVariantAttr<std::string>>();
         }
 
+        if (node["shape"]) {
+            params.shape = node["shape"].as<LayerVariantAttr<std::vector<uint64_t>>> ();
```

Comment:
Why no`size_t` as on `opencv` side?

Reply:
Well size_t is compiler dependent and translates to int64_t. 
When I have to choose when to choose a type between those two this is my thought process:

Taking in consideration that we read from a yaml file the interface is directed towards the end-user who does not care about the compiler so I choose int64_t because it remains the same regardless of the compiler
If the interface was directed towards another api that is aware of the compiler then I would have asked my self.
 1) Is the API compiled with the same compiler then -> size_t
 2) Do I know the to what the compiler translate size_t? YES -> uint64/32_t  or NO -> size_t

Reply:
Also good question. We might have to investigate if it should be int64_t. What happens when we want to define dynamic dimensions that are declared with -1 or "?" in benchmark app.

I will have to look into this

Reply:
What benchmark app calls dynamic shapes is a reshape combined with data_shape. As protopipe does not have data_shape using -1 for -shape(dynamic shapes) is not useful so using int64_t is not needed.

Will change type to size_t

Reply:
Done